### ISSUE TITLE
How to cancel dedicatehost in CLI?

### ISSUE THREAD

**Issue by yhuanghamu** [softlayer/softlayer-python#1041]
**Please triple-check to make sure that you have properly masked out user credentials like usernames, passwords and API keys before submitting your issue**
I know we can cancel it in web portal, but there is no `slcli dedicatedhost cancel` command option.
### Environment Information
Operating System: Mac OSX 10.13.6
softlayer-python version (`slcli --version`): 5.5.1

**Comment by acamacho82** [comment]
Currently there is no way to delete dedicated hosts using the command `slcli dedicatedhost` but you should be able to do it through `slcli call-api`

```
$ slcli call-api Virtual_DedicatedHost deleteObject --id=10201
true
```
To retrieve the list of dedicated host and their ids use `slcli dedicatedhost list`

**Comment by yhuanghamu** [comment]
Nice. Thanks.

**Comment by yhuanghamu** [comment]
Cancellation of Dedicated hosts relied on the cancellation of attached virtual servers. 
Could you add a cascading option? Which means if i am going to cancel a dedicated host then all virtual servers running on top of it will be cancelled as well.
Another feature request is to make it async, which means i can open two tickets, first ticket to cancel virtual servers,  second ticket to cancel dedicated host. Second ticket will be processed after virtual servers are killed.

**Comment by allmightyspiff** [comment]
Merged into master, this will be included in 5.6.4
